.github/workflows/tests.yml

```
@@ -29,17 +29,14 @@ jobs:
-
       - name: Install Node.js
         uses: actions/setup-node@v4
         with:
           node-version: 20
           cache: "npm"
 
-      - run: npm install --frozen-lockfile
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run E2E Tests
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)